repos/system_upgrade/common/actors/peseventsscanner/libraries/peseventsscanner_repomap.py

```diff
@@ -23,7 +23,7 @@ class RepoMapDataHandler(object):
     Provide the basic functionality to work with the repository data easily.
     """
 
-    def __init__(self, repo_map, distro='', cloud_provider='', default_channels=None):
+    def __init__(self, repo_map, src_distro='', dst_distro='', cloud_provider='', default_channels=None):
```

Comment:
Note that elsewhere in the PR we are using `target_distro` whereas here it is `dst_distro`. I would prefer having a single, consistent terminology.